['Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpe7z0d8vk/solution_0.py", line 49, in <module>\n    print(Solution().countDays(days = 10, meetings = [[5,7],[1,3],[9,10]]))\nTypeError: countDays() got an unexpected keyword argument \'days\'\n', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpe7z0d8vk/solution_1.py", line 49, in <module>\n    print(Solution().countDays(days = 5, meetings = [[2,4],[1,3]]))\nTypeError: countDays() got an unexpected keyword argument \'days\'\n']